homeassistant/components/smarthab/translations/fr.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please remove `fr.json`.  The translation system will overwrite anyways.

Reply:
Meaning I have to submit these translations later through Lokalise?

Reply:
Exactly. 👍